charts/artifact-caching-proxy/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 description: artifact-caching-proxy is a Nginx caching proxy in front of repo.jenkins-ci.org
 name: artifact-caching-proxy
 type: application
-version: 0.12.0
+version: 0.12.1
```

Comment:
```suggestion
version: 0.13.0
```

Reply:
Rationale: it's a new feature so minor version bump :)

Reply:
Yeah, makes sense